# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/performance/tests/test_performance_observer.js
# Commit: 3cd097a7b017
# Full Hash: 3cd097a7b017154cf5f7a2d9ed97c7891d3074f4
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-10 09:58:56
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D26654
# ==============================================================================

diff -r ea2baf2c9ae7 -r 3cd097a7b017 dom/performance/tests/test_performance_observer.js
--- a/dom/performance/tests/test_performance_observer.js	Tue Apr 09 20:32:54 2019 +0000
+++ b/dom/performance/tests/test_performance_observer.js	Wed Apr 10 00:14:39 2019 +0000
@@ -12,11 +12,11 @@
   var observer = new PerformanceObserver(() => {
   });
 
-  assert_throws({name: "TypeError"}, function() {
+  assert_throws({name: "SyntaxError"}, function() {
     observer.observe();
   }, "observe() should throw TypeError exception if no option specified.");
 
-  assert_throws({name: "TypeError"}, function() {
+  assert_throws({name: "SyntaxError"}, function() {
     observer.observe({ unsupportedAttribute: "unsupported" });
   }, "obsrve() should throw TypeError exception if the option has no 'entryTypes' attribute.");
 